### PR TITLE
[CI SKIP] Improve wording, manga details new info

### PR DESCRIPTION
**Pull request by AriaMoradi**
continues #4828

I can't push to that branch.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -189,14 +190,18 @@ a.k.a. the Latest source entry point in the app (invoked by tapping on the "Late
...
+
+
 #### Manga Details
 
 - When user taps on a manga, `fetchMangaDetails` and `fetchChapterList` will be called and the results will be cached.
+    - A `SManga` entry is identified by it's `url`.
 - `fetchMangaDetails` is called to update a manga's details from when it was initialized earlier.
     - During a backup, only `url` and `title` are stored. To restore the rest of the manga data, the app calls `fetchMangaDetails`, so all fields should be (re)filled in if possible.
     - `SManga.initialized` tells the app if it should call `fetchMangaDetails`. If you are overriding `fetchMangaDetails`, make sure to pass it as `true`.
+    - If a `SManga` is cached `fetchMangaDetails` will be only called when the user does a manual update(Swipe-to-Refresh).
```
Comment (CarlosEsco) [inline CONTRIBUTING.md:202]: change this from cached to something else like 
If the SManga already exists in the database fetchMangaDetails...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -189,14 +190,18 @@ a.k.a. the Latest source entry point in the app (invoked by tapping on the "Late
     - If search functionality is not available, return `Observable.just(MangasPage(emptyList(), false))`
 - `getFilterList` will be called to get all filters and filter types. **TODO: explain more about `Filter`**
 
+
+
 #### Manga Details
 
 - When user taps on a manga, `fetchMangaDetails` and `fetchChapterList` will be called and the results will be cached.
+    - A `SManga` entry is identified by it's `url`.
```
Comment (CarlosEsco) [inline CONTRIBUTING.md:198]: Maybe 

SManga entries are considered unique by their url.

File: CONTRIBUTING.md (hunk 1 above)
Comment (AriaMoradi) [reply CONTRIBUTING.md:202]: cache is the samething as db here, if we mention db it gets confusing IMO

File: CONTRIBUTING.md (hunk 2 above)
Comment (AriaMoradi) [reply CONTRIBUTING.md:198]: I guess we can consider it resolved?

**Pull request merged**